### PR TITLE
Feature 3.7/memprof testingjs dumper

### PR DESCRIPTION
**Pull request by dothebart**
### Scope & Purpose

Implement facility to acquire memory dumps

- [x] :hankey: test facility
- [ ] :pizza: New feature (requires CHANGELOG entry, feature documentation and release notes)
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification
- [ ] :book: CHANGELOG entry made

### CODE REVIEW

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/13801/

File: README_maintainers.md [hunk 1]
```diff
@@ -259,6 +259,74 @@ symbols are installed, there will be name information about the called procedure
...
+Here is how it is used:
+
+Use this `cmake` option in addition to the normal options:
+
+```
+-DUSE_JEMALLOC_PROF
+```
+
+when building. Make sure you are using the builtin `jemalloc`. You need
+debugging symbols to analyse the heap dumps later on, so compile with
```
Comment (jsteemann) [inline README_maintainers.md:274]:
```suggestion
debugging symbols to analyze the heap dumps later on, so compile with
```

File: README_maintainers.md [hunk 2]
```diff
@@ -259,6 +259,74 @@ symbols are installed, there will be name information about the called procedure
...
+```
+
+When this is done, `jemalloc` internally does some profiling. You can
+then use this endpoint to get a heap dump:
+
+```
+curl "http://localhost:8529/_admin/status?memory=true" > heap.dump
+```
+
+You can analyse such a heap dump with the `jeprof` tool, either by
```
Comment (jsteemann) [inline README_maintainers.md:291]:
```suggestion
You can analyze such a heap dump with the `jeprof` tool, either by
```

**Comment by dothebart** [comment]
replacing by https://github.com/arangodb/arangodb/pull/13473

**Pull request closed without merging**